modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1252 @@
+
+**Spell Checker Pro** 3.2.0 includes the following fixes, additions, and improvements.
+
+////
+To quote my [comment, left on the TinyMCE 6.4 BoM][11] in Notion
+
+> What changes were made to the Enhanced Media Embed plugin? The
+> [changelog][12] contains no entries beyond 2022-09-08 and the only [commits
+> this year][13] have been dependabot bumps or back-end changes.
+
+[11]: <https://notion.so/TinyMCE-6-4-Bill-of-materials-BOM-885a34e3b9884be0b69cd30eba7c618b?d=40e8ab59323f4319bfbb83898a3fe18c#0c32239916564963850ca08c2ab4f112>
+
+[12]: <https://github.com/tinymce/powerspell/blob/main/CHANGELOG.md>
+
+[13]: <https://github.com/tinymce/powerspell/commits/main>
+////
+
+
+For information on the **Spell Checker Pro** plugin, see: xref:introduction-to-tiny-spellchecker.adoc[INtroduction to Spell Checker Pro].
+
+
+=== Advanced Code Editor 3.2.0
+// bforte this is related to https://ephocks.atlassian.net/browse/DOC-1907
+////
+To quote my [comment, left on the TinyMCE 6.4 BoM][11] in Notion
+
+> What changes were made to the Advanced Code Premium plugin? The
+> [changelog][12] contains the below:
+> New button to copy the current dialog content. #TINY-9525
+> New buttons to increase and decrease the font size in the dialog. #TINY-9526
+> New button to toggle dark mode. #TINY-9527
+> New button to toggle fullscreen. #TINY-9529
+
+[14]: <https://github.com/tinymce/advcode/blob/main/CHANGELOG.md>
+
+[15]: <https://github.com/tinymce/advcode/commits/main>
+////
+
+[[accompanying-premium-skins-and-icon-packs-changes]]
+== Accompanying Premium Skins and Icon Packs changes
+
+The {productname} 6.4 release includes an accompanying release of the **Premium Skins and Icon Packs**.
+
+=== Premium Skins and Icon Packs
+
+The **Premium Skins and Icon Packs** release includes the following updates:
+
+The **Premium Skins and Icon Packs** were rebuilt to pull in the changes also incorporated into the default {productname} 6.4 skin, Oxide.
+
+For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].
+
+
+[[improvement]]
+== Improvement
+
+{productname} 6.4 also includes the following improvement:
+
+=== Direct invalid child text nodes of list elements will be wrapped in list item elements.
+
+// https://ephocks.atlassian.net/browse/TINY-4818
+
+Invalid child nodes were throwing a `Uncaught TypeError:` when `<ol>` tags directly included a text node. This `TypeError` would not allow the user to `Tab` a text node while nested within `<ol>/<li>` HTML tags.
+
+Improvements were made to add a content filter for lists within {productname} 6.4. This improvement meant that any _invalid child nodes_ nested within `<li>` elements would be wrapped with this filter to avoid the `Uncaught TypeError:`.
+
+In {productname} 6.4, direct _invalid child text nodes_ of list elements are now wrapped in list item elements, which allow the user to now `Tab` nested text nodes that are within `<ol>/<li>` tags.
```

Comment:
```suggestion
In previous versions of {productname}, when pressing `Tab` while the cursor is placed next to a `<li>` element, a console error would be un expectantly be thrown.

Improvements were made that added a content filter for lists within {productname} 6.4. This improvement meant that any _invalid child nodes_ nested within `<li>` elements would be wrapped with this filter to avoid the `Uncaught TypeError:`.

In {productname} 6.4, pressing `Tab` while the caret is nested within `<li>` tags, no longer throws a error to the console.
```

Reply:
@TheSpyder Please see above suggested changes and improvements to the documentation for this.

Reply:
@TheSpyder moved this to the new review, as the suggestion is outdated.